src/lib/wagmi.ts

```diff
@@ -89,6 +95,61 @@ const WAGMI_SUPPORTED_CHAINS: Chain[] = [
   kava,
 ];
 
+enum ChainId {
+  ETH_MAINNET = '1',
+  ETH_SEPOLIA = '11155111',
+  POLYGON_MAINNET = '137',
+  POLYGON_MUMBAI = '80002',
+  OPT_MAINNET = '10',
+  OPT_SEPOLIA = '11155420',
+  ARB_MAINNET = '42161',
+  ARB_SEPOLIA = '421614',
+  BASE_MAINNET = '8453',
+  BASE_SEPOLIA = '84532',
+}
+
+const getAlchemyRPCUrls = (chainId: ChainId, apiKey: string) => {
+  switch (chainId) {
+    case ChainId.ETH_MAINNET:
+      return `https://eth-mainnet.g.alchemy.com/v2/${apiKey}`;
+    case ChainId.ETH_SEPOLIA:
+      return `https://eth-sepolia.g.alchemy.com/v2/${apiKey}`;
+    case ChainId.POLYGON_MAINNET:
+      return `https://polygon-mainnet.g.alchemy.com/v2/${apiKey}`;
+    case ChainId.POLYGON_MUMBAI:
+      return `https://polygon-amoy.g.alchemy.com/v2/${apiKey}`;
+    case ChainId.OPT_MAINNET:
+      return `https://opt-mainnet.g.alchemy.com/v2/${apiKey}`;
+    case ChainId.OPT_SEPOLIA:
+      return `https://opt-sepolia.g.alchemy.com/v2/${apiKey}`;
+    case ChainId.ARB_MAINNET:
+      return `https://arb-mainnet.g.alchemy.com/v2/${apiKey}`;
+    case ChainId.ARB_SEPOLIA:
+      return `https://arb-sepolia.g.alchemy.com/v2/${apiKey}`;
+    case ChainId.BASE_MAINNET:
+      return `https://base-mainnet.g.alchemy.com/v2/${apiKey}`;
+    case ChainId.BASE_SEPOLIA:
+      return `https://base-sepolia.g.alchemy.com/v2/${apiKey}`;
+    default:
+      return undefined;
+  }
+};
+
+const RPCTransports = [mainnet, ...WAGMI_SUPPORTED_CHAINS].reduce(
```

Comment:
wagmi v2 requires you to define transports (aka define the RPCs) for connector definitions.